### DISCUSSION TITLE
Convite aberto: AOSC Mentorship

### DISCUSSION THREAD

**Discussion by antonio-pedro99** [prest/prest#827] [General] [answered]
Olá, equipe [prest](https://github.com/prest/prest)

Espero que estejam bem. Sou Antonio Pedro da [Angola Open-source Community](https://github.com/angolaosc), uma comunidade de desenvolvedores dedicados a promover o desenvolvimento open-source em Angola.

Estamos atualmente organizando um Programa de Mentoria em Desenvolvimento Open-source em [Angola](https://en.wikipedia.org/wiki/Angola), visando capacitar os nossos desenvolvedores locais e estimular a colaboração com projetos de código aberto de renome internacional.

Gostaríamos de convidar o[prest](https://github.com/prest/prest) para participar como uma organização mentora no nosso programa. Acreditamos que a participação do [prest](https://github.com/prest/prest) como mentor pode ser extremamente valiosa para nossos desenvolvedores, permitindo que eles contribuam ativamente para um projeto relevante e aprendam com as melhores práticas de uma organização experiente.

O nosso [programa](https://github.com/angolaosc/mentorship) terá duração de um mês e contará com diferentes etapas, incluindo pré-mentoria(onde nós a organização, iremos conduzir um workshop introdutório sobre open-source), mentoria, onde os desenvolvedores terão a oportunidade de trabalhar em qualquer issue aberta neste projeto, receber feedback de especialistas e desenvolver as suas habilidades em desenvolvimento de software.

Seria uma honra contar com o [prest](https://github.com/prest/prest) como um mentor no nosso programa. Acreditamos que essa colaboração pode trazer benefícios mútuos, fortalecendo a comunidade open-source em Angola e promovendo a inovação tecnológica em ambos os lados.

Se estiverem interessados em participar ou tiverem-se alguma dúvida, por favor, deixem-nos saber. Estamos ansiosos para uma possível parceria com o [prest](https://github.com/prest/prest).

Agradecemos a sua atenção e consideração.

Atenciosamente,
Antonio Pedro
Angola Open-source Community

**Comment by avelino** [comment]
cc @arxdsilva @vmesel @felipeweb

**Reply by vmesel** [reply]
Acho super válido e acho que consigo dar algumas mentorias com os códigos e ajudar junto se for de interesse dos demais colaboradores do projeto

**Reply by antonio-pedro99** [reply]
Olá @vmesel, muito obrigado pelo interesse. 
Ficariamos muito honrados em tê-lo como mentor. 

Se estiver interessado em um papo para ter mais detalhes, por favor me avise. 

Att

**Reply by felipeweb** [reply]
Oi Antônio, contribuir com o AOSC através do prest será um prazer, conte comigo para ajudar no que precisar. Entra em contato comigo pelo Linkedin ou pelo meu email abaixo que combinamos mais a fundo os detalhes

fpo@felipeweb.dev 
https://www.linkedin.com/in/felipewebcloud

Grato 
Felipe

**Reply by antonio-pedro99** [reply]
@felipeweb Ola, deixei um pedido de conexão no Linkedin. Vou mandar mais informações sobre o nosso programa de mentoria no email. 
Por outra, pode consultar algumas informações gerais aqui: https://mentorship.aosc.social

**Comment by arxdsilva** [comment, marked as answer]
Oi @antonio-pedro99, 

Contribuir com o AOSC através do prest será um prazer, conte comigo para ajudar no que precisar. Entra em contato comigo pelo Linkedin ou pelo meu email abaixo que combinamos mais a fundo os detalhes.

https://www.linkedin.com/in/arxdsilva/
arxdsilva@gmail.com

Att,
Arthur Silva

**Reply by antonio-pedro99** [reply]
Olá @arxdsilva, deixei uma mensagem no LinkedIn

**Comment by vmesel** [comment]
Oi Antonio! Pode entrar em contato comigo via linkedin também!

On Sat, 29 Jul 2023 at 16:28 António Pedro ***@***.***> wrote:

> Olá @arxdsilva <https://github.com/arxdsilva>, deixei uma mensagem no
> LinkedIn
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/prest/prest/discussions/827#discussioncomment-6584309>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABGA2UYFXJHK27GCQZ5PLITXSVP4RANCNFSM6AAAAAA22XTIVY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Atenciosamente,

Vinicius Mesel

**Reply by antonio-pedro99** [reply]
@vmesel deixei uma mensagem também agora.
